contracts/SignatureValidator.sol

```diff
@@ -3,29 +3,57 @@ pragma solidity ^0.4.24;
 
 contract SignatureValidator {
 
-    function doHash(string _message1, uint32 _message2, string _header1, string _header2)
-     pure internal returns (bytes32) {
-        return keccak256(
-            abi.encodePacked(
-                    keccak256(abi.encodePacked(_header1, _header2)),
-                    keccak256(abi.encodePacked(_message1, _message2)))
-        );
+    uint256 constant chainId = 1;
```

Comment:
Is this the mainnet id? If so can we please make it a constructor param so it is easier to deploy for testing on Ropsten?